fuel/app/classes/materia/session/log.php

```diff
@@ -36,6 +36,7 @@ class Session_Log
 	const TYPE_SCORE_ALERT             = 'SCORE_ALERT';
 	const TYPE_SCORE_FEEDBACK          = 'SCORE_FEEDBACK';
 	const TYPE_SCORE_SET_QUESTION      = 'SCORE_SET_QUESTION';
+	const TYPE_UNSCORABLE              = 'SCORE_UNSCORABLE';
```

Comment:
Before this makes it in, I want to ask the group if `SCORE_UNSCORABLE` and `TYPE_UNSCORABLE` are the best labels for this?

I'm thinking  `TYPE_PARTICIPATION_SCORE` and `PARTICIPATION_SCORE` seems more accurate?

Reply:
`TYPE_UNSCORABLE` is now `TYPE_SCORE_PARTICIPATION`
`SCORE_UNSCORABLE` is now `SCORE_PARTICIPATION`